src/main/java/thePackmaster/relics/siegepack/ShellCharger.java

```diff
@@ -24,19 +26,38 @@ public String getUpdatedDescription()
     }
 
     public void atBattleStart() {
-        super.atBattleStart();
         this.setCounter(0);
     }
 
+    public void atTurnStart() {
```

Comment:
what visual bugs? this is weird

Reply:
both relics when added in debug manners seem to not display any number during the fight, most likely because their start methods were never called, until something triggered them. It would fix itself by the next fight. These methods seem to fix the issue within 1 turn but I can remove them, no worries